src/pin_layout.rs

```diff
@@ -1,23 +1,24 @@
-use iced::{Alignment, Color, Element, Length};
+#![allow(unused)] // Need to remove
```

Comment:
Do you want to remove now as part of PR, or later?
What is being unused that it complains about?